docs/generated/packages/nx/executors/run-script.json

```diff
@@ -11,13 +11,15 @@
     "properties": {
       "script": {
         "type": "string",
-        "description": "An npm script name in the `package.json` file of the project (e.g., `build`)."
+        "description": "An npm script name in the `package.json` file of the project (e.g., `build`).",
+        "x-priority": "important"
```

Comment:
`required` properties are going to be `important` by default so don't mark them twice.